docs/templates/module.qmd.jinja2

```diff
@@ -3,7 +3,7 @@
 {% import "macros/navigation.jinja2" as nav %}
 {% import "macros/signatures.jinja2" as signatures %}
 ---
-title: "{% if module.name == "validmind" %}ValidMind Library{% else %}[validmind](/validmind/validmind.qmd).{{ module.name }}{% endif +%}"
+title: "{% if module.name == "validmind" %}ValidMind Library API{% else %}[validmind](/validmind/validmind.qmd).{{ module.name }}{% endif +%}"
```

Comment:
Again, this is not a term we should use. The ValidMind Library happens to have a Python API in it today but it could have other things in it as well in the future and we should likely not reduce the library to the API.